### PR TITLE
[TF] First changes on the road to Keras v3

### PR DESCRIPTION
**Pull request by felixdittrich92**
This PR:

- is part of a series of PR's
- basic elements to go further forward (only .weights.h5 and .keras (full model) - loading possible in keras v3)
- change to keras imports
- **Finally**: enable fine tuning tf models with `pretrained` also if the vocab / classes differs !! :100:  (same we have already a long time ago added to PT - which wasn't possible with loading from `/weights` in TF)

- all models works as expected 

Next steps:

- Implement `build` and `get_config` methods for proper loading & saving (serialization)

Any feedback is welcome :hugs: 

NOTE: It's still fully backwards compatible - but marking as breaking change because that will be the last release where you can load `/weights` - so we have to mention this in the next release notes

@odulcy-mindee models upload :)

### CODE REVIEW

**Comment by felixdittrich92** [comment]
@odulcy-mindee I see two options:

1. We go step by step with smaller PRs until everything works as expected
2. We iterate on this PR and have one giant PR at the end 😅

What do you prefer ?

**Comment by felixdittrich92** [comment]
Next steps:

- try to remove building models first by passing a dummy input -> `build` and `get_config` methods
- test by upgrading to tf >= 2.16 + keras v3 (NOTE: This will be a breaking change -> 1.0.0 - only possible after the next release)
- update training pipeline -> To fix the Dataloader part

**Comment by felixdittrich92** [comment]
#1461

File: references/detection/train_tensorflow.py [hunk 1]
```diff
@@ -184,6 +184,8 @@ def main(args):
 
     # Resume weights
     if isinstance(args.resume, str):
+        # Build the model first to load the weights
+        _ = model(tf.zeros((1, args.input_size, args.input_size, 3)), training=False)
```
Comment (odulcy-mindee) [inline references/detection/train_tensorflow.py:188]: Don't you need to set `training=True` somewhere after `load_weights` ?

File: tests/tensorflow/test_models_utils_tf.py [hunk 2]
```diff
@@ -27,20 +27,18 @@ def test_bf16_to_float32():
 
 
 def test_load_pretrained_params(tmpdir_factory):
-    model = Sequential([layers.Dense(8, activation="relu", input_shape=(4,)), layers.Dense(4)])
+    model = mobilenet_v3_small(pretrained=False)
     # Retrieve this URL
-    url = "https://doctr-static.mindee.com/models?id=v0.1-models/tmp_checkpoint-4a98e492.zip&src=0"
+    url = "https://github.com/mindee/doctr/releases/download/v0.9.0/mobilenet_v3_small-3fcebad7.weights.h5"
```
Comment (odulcy-mindee) [inline tests/tensorflow/test_models_utils_tf.py:32]:
```suggestion
    url = "https://doctr-static.mindee.com/models?id=v0.9.0/mobilenet_v3_small-3fcebad7.weights.h5&src=0"
```

File: doctr/models/recognition/vitstr/tensorflow.py [hunk 3]
```diff
@@ -23,14 +23,14 @@
         "std": (0.299, 0.296, 0.301),
         "input_shape": (32, 128, 3),
         "vocab": VOCABS["french"],
-        "url": "https://doctr-static.mindee.com/models?id=v0.6.0/vitstr_small-358fab2e.zip&src=0",
+        "url": "https://github.com/mindee/doctr/releases/download/v0.9.0/vitstr_small-d28b8d92.weights.h5",
```
Comment (odulcy-mindee) [inline doctr/models/recognition/vitstr/tensorflow.py:26]:
```suggestion
        "url": "https://doctr-static.mindee.com/models?id=v0.9.0/vitstr_small-d28b8d92.weights.h5&src=0",
```

File: doctr/models/recognition/vitstr/tensorflow.py [hunk 4]
```diff
@@ -23,14 +23,14 @@
...
     },
     "vitstr_base": {
         "mean": (0.694, 0.695, 0.693),
         "std": (0.299, 0.296, 0.301),
         "input_shape": (32, 128, 3),
         "vocab": VOCABS["french"],
-        "url": "https://doctr-static.mindee.com/models?id=v0.6.0/vitstr_base-2889159a.zip&src=0",
+        "url": "https://github.com/mindee/doctr/releases/download/v0.9.0/vitstr_base-9ad6eb84.weights.h5",
```
Comment (odulcy-mindee) [inline doctr/models/recognition/vitstr/tensorflow.py:33]:
```suggestion
        "url": "https://doctr-static.mindee.com/models?id=v0.9.0/vitstr_base-9ad6eb84.weights.h5&src=0",
```

File: doctr/models/recognition/sar/tensorflow.py [hunk 5]
```diff
@@ -24,7 +24,7 @@
         "std": (0.299, 0.296, 0.301),
         "input_shape": (32, 128, 3),
         "vocab": VOCABS["french"],
-        "url": "https://doctr-static.mindee.com/models?id=v0.6.0/sar_resnet31-c41e32a5.zip&src=0",
+        "url": "https://github.com/mindee/doctr/releases/download/v0.9.0/sar_resnet31-5a58806c.weights.h5",
```
Comment (odulcy-mindee) [inline doctr/models/recognition/sar/tensorflow.py:27]:
```suggestion
        "url": "https://doctr-static.mindee.com/models?id=v0.9.0/sar_resnet31-5a58806c.weights.h5&src=0",
```

File: doctr/models/recognition/parseq/tensorflow.py [hunk 6]
```diff
@@ -27,7 +27,7 @@
         "std": (0.299, 0.296, 0.301),
         "input_shape": (32, 128, 3),
         "vocab": VOCABS["french"],
-        "url": "https://doctr-static.mindee.com/models?id=v0.6.0/parseq-24cf693e.zip&src=0",
+        "url": "https://github.com/mindee/doctr/releases/download/v0.9.0/parseq-4152a87e.weights.h5",
```
Comment (odulcy-mindee) [inline doctr/models/recognition/parseq/tensorflow.py:30]:
```suggestion
        "url": "https://doctr-static.mindee.com/models?id=v0.9.0/parseq-4152a87e.weights.h5&src=0",
```

File: doctr/models/detection/linknet/tensorflow.py [hunk 7]
```diff
@@ -27,19 +26,19 @@
...
         "input_shape": (1024, 1024, 3),
-        "url": "https://doctr-static.mindee.com/models?id=v0.7.0/linknet_resnet18-b9ee56e6.zip&src=0",
+        "url": "https://github.com/mindee/doctr/releases/download/v0.9.0/linknet_resnet18-615a82c5.weights.h5",
     },
     "linknet_resnet34": {
         "mean": (0.798, 0.785, 0.772),
         "std": (0.264, 0.2749, 0.287),
         "input_shape": (1024, 1024, 3),
-        "url": "https://doctr-static.mindee.com/models?id=v0.7.0/linknet_resnet34-51909c56.zip&src=0",
+        "url": "https://github.com/mindee/doctr/releases/download/v0.9.0/linknet_resnet34-9d772be5.weights.h5",
```
Comment (odulcy-mindee) [inline doctr/models/detection/linknet/tensorflow.py:35]:
```suggestion
        "url": "https://doctr-static.mindee.com/models?id=v0.9.0/linknet_resnet34-9d772be5.weights.h5&src=0",
```

File: doctr/models/detection/fast/tensorflow.py [hunk 8]
```diff
@@ -29,19 +28,19 @@
         "input_shape": (1024, 1024, 3),
         "mean": (0.798, 0.785, 0.772),
         "std": (0.264, 0.2749, 0.287),
-        "url": "https://doctr-static.mindee.com/models?id=v0.8.1/fast_tiny-959daecb.zip&src=0",
+        "url": "https://github.com/mindee/doctr/releases/download/v0.9.0/fast_tiny-d7379d7b.weights.h5",
```
Comment (odulcy-mindee) [inline doctr/models/detection/fast/tensorflow.py:31]:
```suggestion
        "url": "https://doctr-static.mindee.com/models?id=v0.9.0/fast_tiny-d7379d7b.weights.h5&src=0",
```

File: doctr/models/detection/fast/tensorflow.py [hunk 9]
```diff
@@ -29,19 +28,19 @@
...
     },
     "fast_small": {
         "input_shape": (1024, 1024, 3),
         "mean": (0.798, 0.785, 0.772),
         "std": (0.264, 0.2749, 0.287),
-        "url": "https://doctr-static.mindee.com/models?id=v0.8.1/fast_small-f1617503.zip&src=0",
+        "url": "https://github.com/mindee/doctr/releases/download/v0.9.0/fast_small-44b27eb6.weights.h5",
```
Comment (odulcy-mindee) [inline doctr/models/detection/fast/tensorflow.py:37]:
```suggestion
        "url": "https://doctr-static.mindee.com/models?id=v0.9.0/fast_small-44b27eb6.weights.h5&src=0",
```

File: doctr/models/detection/fast/tensorflow.py [hunk 10]
```diff
@@ -29,19 +28,19 @@
...
     },
     "fast_base": {
         "input_shape": (1024, 1024, 3),
         "mean": (0.798, 0.785, 0.772),
         "std": (0.264, 0.2749, 0.287),
-        "url": "https://doctr-static.mindee.com/models?id=v0.8.1/fast_base-255e2ac3.zip&src=0",
+        "url": "https://github.com/mindee/doctr/releases/download/v0.9.0/fast_base-f2c6c736.weights.h5",
```
Comment (odulcy-mindee) [inline doctr/models/detection/fast/tensorflow.py:43]:
```suggestion
        "url": "https://doctr-static.mindee.com/models?id=v0.9.0/fast_base-f2c6c736.weights.h5&src=0",
```

File: doctr/models/detection/differentiable_binarization/tensorflow.py [hunk 11]
```diff
@@ -29,13 +28,13 @@
         "mean": (0.798, 0.785, 0.772),
         "std": (0.264, 0.2749, 0.287),
         "input_shape": (1024, 1024, 3),
-        "url": "https://doctr-static.mindee.com/models?id=v0.7.0/db_resnet50-84171458.zip&src=0",
+        "url": "https://github.com/mindee/doctr/releases/download/v0.9.0/db_resnet50-649fa22b.weights.h5",
```
Comment (odulcy-mindee) [inline doctr/models/detection/differentiable_binarization/tensorflow.py:31]:
```suggestion
        "url": "https://doctr-static.mindee.com/models?id=v0.9.0/db_resnet50-649fa22b.weights.h5&src=0",
```

File: doctr/models/detection/differentiable_binarization/tensorflow.py [hunk 12]
```diff
@@ -29,13 +28,13 @@
...
     },
     "db_mobilenet_v3_large": {
         "mean": (0.798, 0.785, 0.772),
         "std": (0.264, 0.2749, 0.287),
         "input_shape": (1024, 1024, 3),
-        "url": "https://doctr-static.mindee.com/models?id=v0.7.0/db_mobilenet_v3_large-da524564.zip&src=0",
+        "url": "https://github.com/mindee/doctr/releases/download/v0.9.0/db_mobilenet_v3_large-ee2e1dbe.weights.h5",
```
Comment (odulcy-mindee) [inline doctr/models/detection/differentiable_binarization/tensorflow.py:37]:
```suggestion
        "url": "https://doctr-static.mindee.com/models?id=v0.9.0/db_mobilenet_v3_large-ee2e1dbe.weights.h5&src=0",
```

File: doctr/models/classification/vit/tensorflow.py [hunk 13]
```diff
@@ -25,14 +25,14 @@
         "std": (0.299, 0.296, 0.301),
         "input_shape": (3, 32, 32),
         "classes": list(VOCABS["french"]),
-        "url": "https://doctr-static.mindee.com/models?id=v0.6.0/vit_s-6300fcc9.zip&src=0",
+        "url": "https://github.com/mindee/doctr/releases/download/v0.9.0/vit_s-69bc459e.weights.h5",
```
Comment (odulcy-mindee) [inline doctr/models/classification/vit/tensorflow.py:28]:
```suggestion
        "url": "https://doctr-static.mindee.com/models?id=v0.9.0/vit_s-69bc459e.weights.h5&src=0",
```

File: doctr/models/classification/vit/tensorflow.py [hunk 14]
```diff
@@ -25,14 +25,14 @@
...
     },
     "vit_b": {
         "mean": (0.694, 0.695, 0.693),
         "std": (0.299, 0.296, 0.301),
         "input_shape": (32, 32, 3),
         "classes": list(VOCABS["french"]),
-        "url": "https://doctr-static.mindee.com/models?id=v0.6.0/vit_b-57158446.zip&src=0",
+        "url": "https://github.com/mindee/doctr/releases/download/v0.9.0/vit_b-c64705bd.weights.h5",
```
Comment (odulcy-mindee) [inline doctr/models/classification/vit/tensorflow.py:35]:
```suggestion
        "url": "https://doctr-static.mindee.com/models?id=v0.9.0/vit_b-c64705bd.weights.h5&src=0",
```

File: doctr/models/classification/vgg/tensorflow.py [hunk 15]
```diff
@@ -22,7 +22,7 @@
         "std": (1.0, 1.0, 1.0),
         "input_shape": (32, 32, 3),
         "classes": list(VOCABS["french"]),
-        "url": "https://doctr-static.mindee.com/models?id=v0.4.1/vgg16_bn_r-c5836cea.zip&src=0",
+        "url": "https://github.com/mindee/doctr/releases/download/v0.9.0/vgg16_bn_r-b4d69212.weights.h5",
```
Comment (odulcy-mindee) [inline doctr/models/classification/vgg/tensorflow.py:25]:
```suggestion
        "url": "https://doctr-static.mindee.com/models?id=v0.9.0/vgg16_bn_r-b4d69212.weights.h5&src=0",
```

File: doctr/models/classification/textnet/tensorflow.py [hunk 16]
```diff
@@ -22,21 +22,21 @@
         "std": (0.299, 0.296, 0.301),
         "input_shape": (32, 32, 3),
         "classes": list(VOCABS["french"]),
-        "url": "https://doctr-static.mindee.com/models?id=v0.8.1/textnet_tiny-fe9cc245.zip&src=0",
+        "url": "https://github.com/mindee/doctr/releases/download/v0.9.0/textnet_tiny-a29eeb4a.weights.h5",
```
Comment (odulcy-mindee) [inline doctr/models/classification/textnet/tensorflow.py:25]:
```suggestion
        "url": "https://doctr-static.mindee.com/models?id=v0.9.0/textnet_tiny-a29eeb4a.weights.h5&src=0",
```

File: doctr/models/classification/textnet/tensorflow.py [hunk 17]
```diff
@@ -22,21 +22,21 @@
...
     },
     "textnet_small": {
         "mean": (0.694, 0.695, 0.693),
         "std": (0.299, 0.296, 0.301),
         "input_shape": (32, 32, 3),
         "classes": list(VOCABS["french"]),
-        "url": "https://doctr-static.mindee.com/models?id=v0.8.1/textnet_small-29c39c82.zip&src=0",
+        "url": "https://github.com/mindee/doctr/releases/download/v0.9.0/textnet_small-1c2df0e3.weights.h5",
```
Comment (odulcy-mindee) [inline doctr/models/classification/textnet/tensorflow.py:32]:
```suggestion
        "url": "https://doctr-static.mindee.com/models?id=v0.9.0/textnet_small-1c2df0e3.weights.h5&src=0",
```

File: doctr/models/classification/textnet/tensorflow.py [hunk 18]
```diff
@@ -22,21 +22,21 @@
...
     },
     "textnet_base": {
         "mean": (0.694, 0.695, 0.693),
         "std": (0.299, 0.296, 0.301),
         "input_shape": (32, 32, 3),
         "classes": list(VOCABS["french"]),
-        "url": "https://doctr-static.mindee.com/models?id=v0.8.1/textnet_base-168aa82c.zip&src=0",
+        "url": "https://github.com/mindee/doctr/releases/download/v0.9.0/textnet_base-8b4b89bc.weights.h5",
```
Comment (odulcy-mindee) [inline doctr/models/classification/textnet/tensorflow.py:39]:
```suggestion
        "url": "https://doctr-static.mindee.com/models?id=v0.9.0/textnet_base-8b4b89bc.weights.h5&src=0",
```

File: doctr/models/classification/resnet/tensorflow.py [hunk 19]
```diff
@@ -24,35 +24,35 @@
         "std": (0.299, 0.296, 0.301),
         "input_shape": (32, 32, 3),
         "classes": list(VOCABS["french"]),
-        "url": "https://doctr-static.mindee.com/models?id=v0.4.1/resnet18-d4634669.zip&src=0",
+        "url": "https://github.com/mindee/doctr/releases/download/v0.9.0/resnet18-f42d3854.weights.h5",
```
Comment (odulcy-mindee) [inline doctr/models/classification/resnet/tensorflow.py:27]:
```suggestion
        "url": "https://doctr-static.mindee.com/models?id=v0.9.0/resnet18-f42d3854.weights.h5&src=0",
```

File: doctr/models/classification/resnet/tensorflow.py [hunk 20]
```diff
@@ -24,35 +24,35 @@
...
     },
     "resnet31": {
         "mean": (0.694, 0.695, 0.693),
         "std": (0.299, 0.296, 0.301),
         "input_shape": (32, 32, 3),
         "classes": list(VOCABS["french"]),
-        "url": "https://doctr-static.mindee.com/models?id=v0.5.0/resnet31-5a47a60b.zip&src=0",
+        "url": "https://github.com/mindee/doctr/releases/download/v0.9.0/resnet31-ab75f78c.weights.h5",
```
Comment (odulcy-mindee) [inline doctr/models/classification/resnet/tensorflow.py:34]:
```suggestion
        "url": "https://doctr-static.mindee.com/models?id=v0.9.0/resnet31-ab75f78c.weights.h5&src=0",
```

File: doctr/models/classification/resnet/tensorflow.py [hunk 21]
```diff
@@ -24,35 +24,35 @@
...
     },
     "resnet34": {
         "mean": (0.694, 0.695, 0.693),
         "std": (0.299, 0.296, 0.301),
         "input_shape": (32, 32, 3),
         "classes": list(VOCABS["french"]),
-        "url": "https://doctr-static.mindee.com/models?id=v0.5.0/resnet34-5dcc97ca.zip&src=0",
+        "url": "https://github.com/mindee/doctr/releases/download/v0.9.0/resnet34-03967df9.weights.h5",
```
Comment (odulcy-mindee) [inline doctr/models/classification/resnet/tensorflow.py:41]:
```suggestion
        "url": "https://doctr-static.mindee.com/models?id=v0.9.0/resnet34-03967df9.weights.h5&src=0",
```

File: doctr/models/classification/resnet/tensorflow.py [hunk 22]
```diff
@@ -24,35 +24,35 @@
...
     },
     "resnet50": {
         "mean": (0.694, 0.695, 0.693),
         "std": (0.299, 0.296, 0.301),
         "input_shape": (32, 32, 3),
         "classes": list(VOCABS["french"]),
-        "url": "https://doctr-static.mindee.com/models?id=v0.5.0/resnet50-e75e4cdf.zip&src=0",
+        "url": "https://github.com/mindee/doctr/releases/download/v0.9.0/resnet50-82358f34.weights.h5",
```
Comment (odulcy-mindee) [inline doctr/models/classification/resnet/tensorflow.py:48]:
```suggestion
        "url": "https://doctr-static.mindee.com/models?id=v0.9.0/resnet50-82358f34.weights.h5&src=0",
```

File: doctr/models/classification/resnet/tensorflow.py [hunk 23]
```diff
@@ -24,35 +24,35 @@
...
     },
     "resnet34_wide": {
         "mean": (0.694, 0.695, 0.693),
         "std": (0.299, 0.296, 0.301),
         "input_shape": (32, 32, 3),
         "classes": list(VOCABS["french"]),
-        "url": "https://doctr-static.mindee.com/models?id=v0.5.0/resnet34_wide-c1271816.zip&src=0",
+        "url": "https://github.com/mindee/doctr/releases/download/v0.9.0/resnet34_wide-b18fdf79.weights.h5",
```
Comment (odulcy-mindee) [inline doctr/models/classification/resnet/tensorflow.py:55]:
```suggestion
        "url": "https://doctr-static.mindee.com/models?id=v0.9.0/resnet34_wide-b18fdf79.weights.h5&src=0",
```

File: doctr/models/classification/magc_resnet/tensorflow.py [hunk 24]
```diff
@@ -26,7 +26,7 @@
         "std": (0.299, 0.296, 0.301),
         "input_shape": (32, 32, 3),
         "classes": list(VOCABS["french"]),
-        "url": "https://doctr-static.mindee.com/models?id=v0.6.0/magc_resnet31-addbb705.zip&src=0",
+        "url": "https://github.com/mindee/doctr/releases/download/v0.9.0/magc_resnet31-16aa7d71.weights.h5",
```
Comment (odulcy-mindee) [inline doctr/models/classification/magc_resnet/tensorflow.py:29]:
```suggestion
        "url": "https://doctr-static.mindee.com/models?id=v0.9.0/magc_resnet31-16aa7d71.weights.h5&src=0",
```

File: doctr/models/classification/mobilenet/tensorflow.py [hunk 25]
```diff
@@ -32,42 +32,42 @@
         "std": (0.299, 0.296, 0.301),
         "input_shape": (32, 32, 3),
         "classes": list(VOCABS["french"]),
-        "url": "https://doctr-static.mindee.com/models?id=v0.4.1/mobilenet_v3_large-47d25d7e.zip&src=0",
+        "url": "https://github.com/mindee/doctr/releases/download/v0.9.0/mobilenet_v3_large-d857506e.weights.h5",
```
Comment (odulcy-mindee) [inline doctr/models/classification/mobilenet/tensorflow.py:35]:
```suggestion
        "url": "https://doctr-static.mindee.com/models?id=v0.9.0/mobilenet_v3_large-d857506e.weights.h5&src=0",
```

File: doctr/models/classification/mobilenet/tensorflow.py [hunk 26]
```diff
@@ -32,42 +32,42 @@
...
     },
     "mobilenet_v3_large_r": {
         "mean": (0.694, 0.695, 0.693),
         "std": (0.299, 0.296, 0.301),
         "input_shape": (32, 32, 3),
         "classes": list(VOCABS["french"]),
-        "url": "https://doctr-static.mindee.com/models?id=v0.4.1/mobilenet_v3_large_r-a108e192.zip&src=0",
+        "url": "https://github.com/mindee/doctr/releases/download/v0.9.0/mobilenet_v3_large_r-eef2e3c6.weights.h5",
```
Comment (odulcy-mindee) [inline doctr/models/classification/mobilenet/tensorflow.py:42]:
```suggestion
        "url": "https://doctr-static.mindee.com/models?id=v0.9.0/mobilenet_v3_large_r-eef2e3c6.weights.h5&src=0",
```

File: doctr/models/classification/mobilenet/tensorflow.py [hunk 27]
```diff
@@ -32,42 +32,42 @@
...
     },
     "mobilenet_v3_small": {
         "mean": (0.694, 0.695, 0.693),
         "std": (0.299, 0.296, 0.301),
         "input_shape": (32, 32, 3),
         "classes": list(VOCABS["french"]),
-        "url": "https://doctr-static.mindee.com/models?id=v0.4.1/mobilenet_v3_small-8a32c32c.zip&src=0",
+        "url": "https://github.com/mindee/doctr/releases/download/v0.9.0/mobilenet_v3_small-3fcebad7.weights.h5",
```
Comment (odulcy-mindee) [inline doctr/models/classification/mobilenet/tensorflow.py:49]:
```suggestion
        "url": "https://doctr-static.mindee.com/models?id=v0.9.0/mobilenet_v3_small-3fcebad7.weights.h5&src=0",
```

File: doctr/models/classification/mobilenet/tensorflow.py [hunk 28]
```diff
@@ -32,42 +32,42 @@
...
     },
     "mobilenet_v3_small_r": {
         "mean": (0.694, 0.695, 0.693),
         "std": (0.299, 0.296, 0.301),
         "input_shape": (32, 32, 3),
         "classes": list(VOCABS["french"]),
-        "url": "https://doctr-static.mindee.com/models?id=v0.4.1/mobilenet_v3_small_r-3d61452e.zip&src=0",
+        "url": "https://github.com/mindee/doctr/releases/download/v0.9.0/mobilenet_v3_small_r-dd50218d.weights.h5",
```
Comment (odulcy-mindee) [inline doctr/models/classification/mobilenet/tensorflow.py:56]:
```suggestion
        "url": "https://doctr-static.mindee.com/models?id=v0.9.0/mobilenet_v3_small_r-dd50218d.weights.h5&src=0",
```

File: doctr/models/classification/mobilenet/tensorflow.py [hunk 29]
```diff
@@ -32,42 +32,42 @@
...
     },
     "mobilenet_v3_small_crop_orientation": {
         "mean": (0.694, 0.695, 0.693),
         "std": (0.299, 0.296, 0.301),
         "input_shape": (128, 128, 3),
         "classes": [0, -90, 180, 90],
-        "url": "https://doctr-static.mindee.com/models?id=v0.4.1/classif_mobilenet_v3_small-1ea8db03.zip&src=0",
+        "url": "https://github.com/mindee/doctr/releases/download/v0.9.0/mobilenet_v3_small_crop_orientation-ef019b6b.weights.h5",
```
Comment (odulcy-mindee) [inline doctr/models/classification/mobilenet/tensorflow.py:63]:
```suggestion
        "url": "https://doctr-static.mindee.com/models?id=v0.9.0/mobilenet_v3_small_crop_orientation-ef019b6b.weights.h5&src=0",
```

File: doctr/models/classification/mobilenet/tensorflow.py [hunk 30]
```diff
@@ -32,42 +32,42 @@
...
     },
     "mobilenet_v3_small_page_orientation": {
         "mean": (0.694, 0.695, 0.693),
         "std": (0.299, 0.296, 0.301),
         "input_shape": (512, 512, 3),
         "classes": [0, -90, 180, 90],
-        "url": "https://doctr-static.mindee.com/models?id=v0.8.1/mobilenet_v3_small_page_orientation-aec9553e.zip&src=0",
+        "url": "https://github.com/mindee/doctr/releases/download/v0.9.0/mobilenet_v3_small_page_orientation-0071d55d.weights.h5",
```
Comment (odulcy-mindee) [inline doctr/models/classification/mobilenet/tensorflow.py:70]:
```suggestion
        "url": "https://doctr-static.mindee.com/models?id=v0.9.0/mobilenet_v3_small_page_orientation-0071d55d.weights.h5&src=0",
```

**Review by odulcy-mindee — CHANGES_REQUESTED** [review]

File: pyproject.toml [hunk 31]
```diff
@@ -56,6 +56,8 @@ tf = [
     # cf. https://github.com/mindee/doctr/pull/1461
     "tensorflow>=2.11.0,<2.16.0",
     "tf2onnx>=1.16.0,<2.0.0",  # cf. https://github.com/onnx/tensorflow-onnx/releases/tag/v1.16.0
+    # TODO: This is a temporary fix until we can upgrade to a newer version of tensorflow
```
Comment (felixdittrich92) [inline pyproject.toml:59]: Temp pin numpy until we can upgrade tensorflow to fix the issue with `poetry` and the failing API test (Will have effect after merging)

**Review by odulcy-mindee — APPROVED** [review]

**Pull request merged**